### PR TITLE
Update getting-started.md with Bandit

### PR DESCRIPTION
**Pull request by pdgonzalez872**

### CODE REVIEW

File: content/en/docs/languages/erlang/getting-started.md [hunk 1]
```diff
@@ -59,7 +65,10 @@ def deps do
     {:opentelemetry_api, "~> {{% param versions.otelApi %}}"},
     {:opentelemetry_exporter, "~> {{% param versions.otelExporter %}}"},
     {:opentelemetry_phoenix, "~> {{% param versions.otelPhoenix %}}"},
-    {:opentelemetry_cowboy, "~> {{% param versions.otelCowboy %}}"},
+    # for Cowboy
+    {:opentelemetry_cowboy, "~> {{% param versions.otelCowboy %}}"}
+    # for Bandit
+    {:opentelemetry_bandit, "~> 0.1.4"},
```
Comment (pdgonzalez872) [inline content/en/docs/languages/erlang/getting-started.md:71]: hmm, this doesn't seem correct. Will wait for a review here.

**Comment by svrnm** [comment]
@pdgonzalez872 thanks for raising this PR! Can you provide some details on the background for this PR? From my point of view the getting started should only make use of one webserver and then exercise the whole example with it. In other languages we have a section at the top that calls out that instrumentation is available for other languages, we can add this for sure.

**Comment by tsloughter** [comment]
@svrnm The framework Phoenix only in the last release changed its default to the web server Bandit. So I think this was meant to cover the case for users on the latest Phoenix as well as still on previous versions who are looking to add OpenTelemetry.

Maybe just leaning in on the latest makes sense though.

**Comment by svrnm** [comment]
Thanks @tsloughter that helps! Then this change makes sense.

**Comment by pdgonzalez872** [comment]
@tsloughter is spot on.

I'd still try to have both the old and the new web servers as options if possible. Even though it's the new default, it is still somewhat rare. The docs, especially, the getting started, should help folks hit the ground running, that's why I went the old/new approach.

WDYT?

**Review by svrnm — APPROVED** [review]

**Review by tsloughter — APPROVED** [review]

**Comment by pdgonzalez872** [comment]
hey, that's a cool check! 
image


fixed

**Comment by bryannaegele** [comment]
@pdgonzalez872 can we please get this PR reverted? These updates were premature and should not have been approved. Apologies for the late request.

**Comment by pdgonzalez872** [comment]
@bryannaegele absolutely. I tried creating a revert automatically, but it didn't like it:
image

Has this been done already?

**Pull request merged**